src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -114,7 +114,7 @@ public partial class IsolatedStorageFileStream : System.IO.FileStream
         public override bool CanWrite { get { throw null; } }
         // https://github.com/dotnet/corefx/issues/11127
         // [System.ObsoleteAttribute("This property has been deprecated.  Please use IsolatedStorageFileStream's SafeFileHandle property instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
-        public override System.IntPtr Handle { get { throw null; } }
+        // public override System.IntPtr Handle { get { throw null; } }
```

Comment:
The rest of the change is good to go, I've updated and will commit when green again.